system-contracts/contracts/EvmInterpreterLoop.template.yul

```diff
@@ -528,7 +526,7 @@ for { } true { } {
 
         offset, sp := popStackItem(sp)
 
-        let expansionGas := expandMemory(add(offset, 32))
+        let expansionGas := expandMemory(offset) // TODO: add +32 here
```

Comment:
Are this changes in order to be "gas-compatible" with solidity?

Reply:
Yes